working-together/README.md

```diff
@@ -0,0 +1,23 @@
+Working Together
+==============
+
+A guide for working well together.
+
+
+Pair Programming
+-------
+
+**Before:**
+* Agree Scope (Pairing for two hours? Until the ticket is complete? Just get pass the bug?)
+* Agree Working Environment (Two keyboards? VIM/what other tools?)
+* Agree Pairing Style (Time-based? Ping pong? To the next traffic light?)
```

Comment:
Just curious, is the "next traffic light" similar to the "ping pong" pattern?

Reply:
As in the TDD "red, green, refactor" traffic lights... Sounds like I might just have made this up...

Reply:
Perhaps it's just a pairing pattern that I'm not familiar with. Or perhaps you have invented a new style :)  I like the name. It seems very similar to ping-pong pairing but this approach changes drivers when refactoring? I believe the refactoring stage is not explicitly called out in the ping-pong pattern.

How I'm imagining "to the next traffic light" pairing:
A writes a new test and sees that it fails (red)
B implements the code needed to pass the test (green)
A refactors (refactor)
B writes the next test and sees that it fails (red light)
... etc ...